### ISSUE TITLE
Device not recognized

### ISSUE THREAD

**Issue by raeberli** [varzac/EdgeDrivers#8]
Hi,

Thanks for you work. I was searching for a driver to use the Aqara TVOC Air Quality Monitor (AAQS-S01) and found your driver. I installed it on my hub but every time I got only a "Thing" found by Smartthings. I saw another Driver from someone else that was working but there was the TVOC value not implemented. 

I compared the fingerprints.yml and the working one did not had the LUMI/ prefix on the IDs. So I downloaded your code, removed the prefix from the fingerprints.yml and installed it on my hub. Added the device and it was not using you driver and I see all the values. 

What did I wrong that the driver is working for you, but not form me without this change? My Hub is v2 with FW 000.040.00006.

Best Regards,
Reto

**Comment by Christianterp** [comment]
I have the issue with this driver and have the same setup

**Comment by varzac** [comment]
Sorry about the slow response.  I am not sure what the issue might be.  Can you link to the other driver you found that worked?  The ID of the fingerprint should not affect the way the device joins at all.  It should only depend on the manufacturer and model reported by the device when it joins.

**Comment by raeberli** [comment]
Strange, I tried multiple times to install your driver as I first expected it to conflict with the smartthings edge drivers active. But even with just your driver active I got only a Thing after discovery.

I then tested https://github.com/DJFliX/smartthings-edge-drivers and with this one I got the device recognized as aqara device. 

I only used the dev tools to install your updated driver to my hub and don't know how the public publish works. Could it be that over the invite link I don't get the latest version? It seems that the air quality monitor is the last update you made.
